api/v2alpha1/pipeline_spec.proto

```diff
@@ -669,15 +620,13 @@ message PipelineDeploymentConfig {
     // The query to fetch artifacts.
     message ArtifactQuerySpec {
       // The filter of the artifact query. The supported syntax are:
-      // - `contexts.name='<context name>'`
+      // - `in_context("<context name>")`
       // - `artifact_type='<artifact type name>'`
       // - `uri='<uri>'`
       // - `state=<state>`
-      // - `properties['key']='value'`

```

Comment:
Wouldn't the equivalent metadata fields be supported.

So the existing ones `properties['key']='value'` will be `metadata.key.<type-name> = value` and `custom_properties['key']='value'` will be `metadata.key.<type-name> = value`

Reply:
ack.